src/lib/list/selection-list.spec.ts

```diff
@@ -626,6 +627,23 @@ describe('MatSelectionList without forms', () => {
 
       expect(selectList.selected.length).toBe(0);
     });
+
+    it('should update state of options if list state has changed', () => {
+      // To verify that the template of the list options has been re-rendered after the disabled
+      // property of the selection list has been updated, the ripple directive can be used.
+      // Inspecting the host classes of the options doesn't work because those update as part
```

Comment:
I'm not sure I get this part. Both the disabled class/aria-disabled and the ripple depend on the same property, which means that they should update at the same time, if they don't it could indicate that the test is catching another issue.

Reply:
The difference is: The host bindings of the `<mat-list-option>` seem to be updated as part of the parent template (`<mat-selection-list>`), while the actual template of the `<mat-list-option>` is only being updated if the change detection runs on `MatListOption` (which is obviously not the case)

Maybe I miss something crucial, but from my findings it looks like this is the case.